custom_components/tuya_local/devices/cleanair_605b_humidifier.yaml

```diff
@@ -0,0 +1,133 @@
+name: Clean Air 605B humidifier
+primary_entity:
+  entity: humidifier
+  dps:
+    - id: 1
+      name: switch
+      type: boolean
+    - id: 13
+      name: humidity
+      type: integer
+      range:
+        min: 30
+        max: 75
+        scale: 1
+        step: 5
+      unit: "%"
+    - id: 14
+      name: current_humidity
+      type: integer
+      readonly: true
+      unit: "%"
+    - id: 23
+      name: mode
+      type: string
+      mapping:
+        - dps_val: auto
+          value: auto
+        - dps_val: level1
+          value: "1"
+        - dps_val: level2
+          value: "2"
+        - dps_val: level3
+          value: "3"
+secondary_entities:
+  - entity: sensor
+    name: Current temperature
+    class: temperature
+    dps:
+      - id: 10
+        type: integer
+        name: sensor
+        readonly: true
+        unit: C
+  - entity: switch
+    name: Sleep mode
+    dps:
+      - id: 16
+        type: boolean
+        name: switch
+  - entity: select
+    name: Timer
+    dps:
+      - id: 19
+        type: string
+        name: option
+        mapping:
+          - dps_val: cancel
+            value: no timer active
+          - dps_val: 1h
+            value: 1h
+          - dps_val: 2h
+            value: 2h
+          - dps_val: 3h
+            value: 3h
+          - dps_val: 4h
+            value: 4h
+          - dps_val: 5h
+            value: 5h
+          - dps_val: 6h
+            value: 6h
+          - dps_val: 7h
+            value: 7h
+          - dps_val: 8h
+            value: 8h
+          - dps_val: 9h
+            value: 9h
+          - dps_val: 10h
+            value: 10h
+          - dps_val: 11h
+            value: 11h
+          - dps_val: 12h
+            value: 12h
+  - entity: sensor
+    name: Timer minutes left
+    class: duration
+    dps:
+      - id: 20
+        type: integer
+        name: sensor
+        readonly: true
+        unit: "min"
+  - entity: binary_sensor
+    name: Water tank empty
+    class: problem
+    category: diagnostic
+    dps:
+      - id: 22
+        type: bitfield
+        name: sensor
+        readonly: true
+        mapping:
+          - dps_val: 0
+            value: false
+          - value: true
+  - entity: switch
+    name: Negative ions
+    dps:
+      - id: 25
+        type: boolean
+        name: switch
+  - entity: binary_sensor
+    name: Filter reset 
```

Comment:
"Filter reset" implies a button, that needs to be pushed after the filter is changed, rather than a binary_sensor which would be redundant as "Filter life" is providing the same info.

Reply:
good catch, didn't notice the attribute was read-writable. Implemented as button now.

Reply:
I think it could still be marked as `category: config`, because it isn't really an everyday control everyone would want on their dashboard.

Reply:
moved to category config